### DISCUSSION TITLE
More updated release?

### DISCUSSION THREAD

**Discussion by Mitch2025** [trumank/mint#206] [General]
I've been having issues with the MS Store version and many mods I use (Weapon Heat Crosshair, Death Marker, Alternate PlayerUI, Better Explosion Range Indicator, Combined Resources) not saving their configs in when I make changes to them in modhub or nothing happening at all when making a change. I found this issue https://github.com/trumank/mint/issues/86 which sounded similar to my issue and noticed the comment saying a fix was implemented but that was dated about a week after the current release meaning the current downloadable release doesn't have that fix or any fixes/improvements since Aug 18th, 2023. I was able to figure out how to compile the latest version and it fixed all my mod config issues I've been having with the MS Store version using the current release. I think it'd be a good idea to update the current release to have all the newest fixes as I can see the average user struggling to figure out how to clone and compile it themselves.

**Comment by bluez-dev** [comment]
I've also seen people struggling with the now-outdated release. I'm not sure why there's not a more often release, but I think it would be smart to have at least some kind of regularly updated release. If the current state is not up to standard for a release, maybe a pre-release/unstable branch..? Or upgrade it to a new Minor version because of the new features?

**Comment by baixiongnull** [comment]
I've also seen discussions of new versions, but the release list is still updated a year ago. Why not release the new version that has been fixed, or at least tell us where to get the latest version.

**Comment by baixiongnull** [comment]
I see that the author has seen this question and added a tag. Why doesn't the author reply?

**Comment by Mitch2025** [comment]
For anyone who wants the newer version with all the recent fixes and updates, I've cloned and compiled it on my own github page here:

https://github.com/Mitch2025/mint/releases

All I did I did was fork it and fumbled my way around github actions until I figured out how to use a workflow to build it. Wish I would have figured out this way before since it was way easier than installing git and all the pre-req stuff to clone and compile it on my own PC like I was doing.

**Reply by ORLOX93** [reply]
You called it "v0.2.10Beta - Unoffical" but v0.2.10 IS the current version, that is a bit confusing, advace a number ( like v0.2.11Beta - Unoffical or somehting like that)

Also how the hell do you compile a version? I have always seen people saying "compile it yourself if you dont trust it" and have never seen anyone explaing beyond that. If it were that easy they would compile it for you, but they never do (This is a weird exception, which i appreciate).

EDIT: Oh! The new Mint is so beautiful, it event has colors on the console.

**Reply by Mitch2025** [reply]
I didn't want to mess with the versioning since it's not my software and 0.2.10 is still what the code itself says. I really just wanted to make it clear that it was an unoffical pre-release.

As for compiling, honestly, this is the only thing I've ever compiled and I don't know the technical side of it and just looked up guides. I saw github said MINT was coded in Rust so I googled how to compile rust code from github and went from there. Ended up looking at a few different guides and documentation to find scripts and commands needed to do it. But unless you already have a pretty good understanding of how to use command line and scripts, it could be difficult to understand which is why I always disliked that stance to "just compile it yourself". Not everyone has an IT background to figure that out.

**Reply by ORLOX93** [reply]
@Mitch2025 do you know why your mint keeps trying to fetch files for deleted mods?

Some of my mods got either hidden or deleted, and when i try to update cache i get an error from those mods.
I tred disabling them but the error persisted, so then i deleted the mods, But it's still trying to update them.

Do you know how could i fix this? i currently cant update anything

EDIT: It looks like i somehow fixed the original Mint and can update using it, but i have more than 90 mods and updating there takes like 10~15 minutes. I really like updating using your Mint (its faster) and would like to get it fixed.

**Reply by Mitch2025** [reply]
Unfortunately I have no idea. This isn't "my" mint. I didn't alter or add any code or even know how to code. I just cloned trumanks updated MINT code and told github to compile it into an windows program by reading a couple guides.

**Reply by ORLOX93** [reply]
@Mitch2025 ah... and you arent having the problem i describe right now?

**Reply by yamei333** [reply]
@ORLOX93 
Once a mod is added, it will be checked when updating (even if you have deleted it from the profile).
Currently you can only solve this problem by editing cache.json. (Delete related module information)
Fortunately, "v0.2.10Beta - Unoffical" will already prompt you which module has an error when updating.

sry for google translate

**Reply by ORLOX93** [reply]
@yamei333 i swear i tried that, and i even tried deleting the entire cache thing. It worked on the original Mint, but not on this one.

EDIT: i found out the issue. Whenever i delete lines from cache.json... Mint adds them back when i close it...
How do I fix that? 😅

EDIT2: I Fixed it. Turns out the first paragraph of cache.json is called "ModioCache", and this is where it stores all the mod that it's going to update. I WAS deleteting the mod modules, but i wasnt deleting the entries there, now it works again.
